x-pack/legacy/plugins/uptime/public/components/functional/ping_list/ping_list.tsx

```diff
@@ -220,120 +246,98 @@ export const PingListComponent = ({
         return (
           <EuiButtonIcon
             onClick={() => toggleDetails(item, itemIdToExpandedRowMap, setItemIdToExpandedRowMap)}
-            disabled={!item.error && !(item.http?.response?.body?.bytes > 0)}
+            disabled={!item.error && !(item.http?.response?.body?.content_bytes ?? 0 > 0)}
```

Comment:
any idea, why this is changed?

Reply:
Good point - it should probably stay `bytes` rather than `content_bytes` so they see the full size of the content. Will change back.

Reply:
This was fixed in https://github.com/elastic/kibana/pull/59392/commits/1df28d66e4cf2e0a6217cca8d37ed8b3dfbcf2da, among a bunch of other fixes/improvements.